### PR TITLE
Clickable search results

### PR DESCRIPTION
**Pull request by Aprillion**
Screenshot 2022-06-29 at 18 20 36

### CODE REVIEW

File: app/stampy.ts [hunk 1]
```diff
@@ -57,16 +57,18 @@ const getAnswer = (html: string): string => {
...
-      ?.parentNode.nextElementSibling.querySelectorAll('li')
-      .map((li) => li.querySelector('a:not(.new)')?.innerText) // ignore red links with class new
-      .filter(typedBoolean) ?? []
-  return links
+  const links = root
+    .querySelector('#Related_questionsedit')
+    ?.parentNode.nextElementSibling.querySelectorAll('li')
+    .map((li) => ({
+      title: li.querySelector('a:not(.new)')?.innerText ?? '', // ignore red links with class new
+      pageid: Number(li.querySelector('div[style*="display: none"]')?.innerText),
```
Comment (Aprillion) [inline app/stampy.ts:68]: @plexish using the div with display none as trusted source of pageid now

**Review by ccstan99 — COMMENTED** [review]
Looks good! There's some quirkiness (not consistent) when selecting a question already on page. Some of it's related to canonical questions that don't have canonical answers. Then selected it again from the dropdown. Sometimes the quester/answer gets swapped out unexpectedly or gets stuck in loading state. I'll try testing a bit more later to give reproducible steps.

**Comment by Aprillion** [comment]
yup, there might be bugs, let's find out repro steps and try to fix later

**Review by ccstan99 — APPROVED** [review]
Approved!

**Pull request merged**